src/SFA.DAS.LevyTransferMatching.Api.UnitTests/Controllers/PledgeTests/GetPledgeApplicationsTests.cs

```diff
@@ -45,6 +45,8 @@ public async Task GetApplications_Returns_GetApplicationsResponse()
             Assert.IsNotNull(okObjectResult);
             Assert.IsNotNull(response);
             Assert.AreEqual(_queryResult.Applications.Count(), response.Applications.Count());
+            Assert.IsTrue(response.PledgeRemainingAmount >= 0);
```

Comment:
We could assert equality with the corresponding source value in `_queryResult`.

Reply:
Done now.